### PR TITLE
Increased Websocket Timeout

### PR DESCRIPTION
**Pull request by ermalsh**
From 4 seconds to 15 seconds.

### CODE REVIEW

**Comment by benawad** [comment]
this timeout is fine, we need to increase the one for ReconnectingWebsocket

**Comment by ermalsh** [comment]
> this timeout is fine, we need to increase the one for ReconnectingWebsocket

Gotcha, any preference to the increase amount? Default is 4000ms.

**Comment by benawad** [comment]
do 15 seconds

File: kofta/src/createWebsocket.ts [hunk 1]
```diff
@@ -29,9 +29,39 @@ export const createWebSocket = () => {
...
+    maxRetries: Infinity,
+    maxEnqueuedMessages: Infinity,
+    startClosed: false,
+    debug: false,
+  */
+  const wsOpts = {
+    connectionTimeout: 15000 //15 seconds
+  };
 
+  ws = new ReconnectingWebSocket(apiBaseUrl.replace("http", "ws") + "/socket", [], wsOpts);
```
Comment (benawad) [inline kofta/src/createWebsocket.ts:62]: just pass the option directly in and remove the docs

**Pull request closed without merging**